crypto/kdf/hkdf.c

```diff
@@ -150,25 +151,25 @@ static int pkey_hkdf_ctrl_str(EVP_PKEY_CTX *ctx, const char *type,
 
     if (strcmp(type, "md") == 0)
         return EVP_PKEY_CTX_md(ctx, EVP_PKEY_OP_DERIVE,
-                               EVP_PKEY_CTRL_HKDF_MD, value);
+                               EVP_PKEY_CTRL_PBE_MD, value);
```

Comment:
It is :-/

Typical "oh, andj ust this one more occurrence, then commit" syndrome. Sorry about that.